Source/WarpX.cpp

```diff
@@ -123,6 +123,7 @@ bool WarpX::do_divb_cleaning = false;
 int WarpX::em_solver_medium;
 int WarpX::macroscopic_solver_algo;
 bool WarpX::do_single_precision_comms = false;
+bool WarpX::do_shared_mem_charge_deposition = true;
```

Comment:
This should be unset, or set based on platform and performance measurements, before merging